lib/rex/post/meterpreter/extensions/stdapi/railgun/def/windows/def_wldap32.rb

```diff
@@ -42,17 +42,17 @@ def self.create_library(constant_manager, library_path = 'wldap32')
     ], 'ldap_set_option', "cdecl")
 
     dll.add_function('ldap_search_ext_sA', 'DWORD',[
-        ['DWORD', 'ld', 'in'],
+        ['PHANDLE', 'ld', 'in'],
         ['PCHAR', 'base', 'in'],
-        ['DWORD', 'scope', 'in'],
+        ['ULONG', 'scope', 'in'],
         ['PCHAR', 'filter', 'in'],
         ['PCHAR', 'attrs[]', 'in'],
-        ['DWORD', 'attrsonly', 'in'],
-        ['DWORD', 'pServerControls', 'in'],
-        ['DWORD', 'pClientControls', 'in'],
-        ['DWORD', 'pTimeout', 'in'],
-        ['DWORD', 'SizeLimit', 'in'],
-        ['PDWORD', 'res', 'out']
+        ['ULONG', 'attrsonly', 'in'],
+        ['PVOID', 'pServerControls', 'in'],
+        ['PVOID', 'pClientControls', 'in'],
```

Comment:
`PVOID` isn't a type Railgun allows. It needs to be `LPVOID` because of how the `P` prefix is handled.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/17562/commits/7e705eb7e6c3f2d2a48d1ce193fd2cb1c9859032